From https://github.com/Shaikh-Ubaid/lpython-conda-testing/actions/runs/6282537317/job/17062084700?pr=1,

```bash
semantic error: Function 'field' is not declared and not intrinsic
 --> /home/runner/work/lpython-conda-testing/lpython-conda-testing/lpython/integration_tests/array_expr_03.py:8:16
  |
8 |     a: i8[4] = field(default_factory=lambda: empty(4, dtype=int8))
  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 


Note: if any of the above error or warning messages are not clear or are lacking
context please report it to us (we consider that a bug that must be fixed).
semantic error: array accepts only 1 argument for now, got 3107819[61](https://github.com/Shaikh-Ubaid/lpython-conda-testing/actions/runs/6282537317/job/17062084700?pr=1#step:7:62)94145634 arguments instead.
[  1%] Built target variable_decl_02
 --> /home/runner/work/lpython-conda-testing/lpython-conda-testing/lpython/integration_tests/array_expr_04.py:5:17
  |
5 |     a8: i8[4] = array([127, -127, 3, 111], dtype=int8)
  |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 


Note: if any of the above error or warning messages are not clear or are lacking
context please report it to us (we consider that a bug that must be fixed).
gmake[2]: *** [CMakeFiles/array_expr_03.dir/build.make:74: array_expr_03.o] Error 2
[  1%] Built target array_expr_01
gmake[1]: *** [CMakeFiles/Makefile2:936: CMakeFiles/array_expr_03.dir/all] Error 2
gmake[1]: *** Waiting for unfinished jobs....
gmake[2]: *** [CMakeFiles/array_expr_04.dir/build.make:74: array_expr_04.o] Error 2
gmake[1]: *** [CMakeFiles/Makefile2:9[62](https://github.com/Shaikh-Ubaid/lpython-conda-testing/actions/runs/6282537317/job/17062084700?pr=1#step:7:63): CMakeFiles/array_expr_04.dir/all] Error 2
[  1%] Generating array_expr_06.o
[  1%] Generating array_expr_05.o
[  1%] Built target array_expr_02
semantic error: array accepts only 1 argument for now, got 31078196194145[63](https://github.com/Shaikh-Ubaid/lpython-conda-testing/actions/runs/6282537317/job/17062084700?pr=1#step:7:64)4 arguments instead.
 --> /home/runner/work/lpython-conda-testing/lpython-conda-testing/lpython/integration_tests/array_expr_06.py:5:19
  |
5 |     a32: f32[4] = array([127, -127, 3, 111], dtype=float32)
  |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

I think at the moment we need to make a new release with the latest LPython.

--- a/.github/workflows/CI.yml
+++ b/.github/workflows/CI.yml
@@ -39,7 +39,7 @@ jobs:
         shell: bash -e -l {0}
         run: |
             git clone https://github.com/lcompilers/lpython.git
-            git checkout 2293972e5ce377787d134541f8363ae441030b76
+            git checkout v0.20.0
 
       - name: Run LPython Ctest
         shell: bash -e -l {0}

--- a/.github/workflows/CI.yml
+++ b/.github/workflows/CI.yml
@@ -39,6 +39,7 @@ jobs:
         shell: bash -e -l {0}
         run: |
             git clone https://github.com/lcompilers/lpython.git
+            git checkout 2293972e5ce377787d134541f8363ae441030b76
 
       - name: Run LPython Ctest
         shell: bash -e -l {0}

--- a/environment.yml
+++ b/environment.yml
@@ -4,6 +4,6 @@ channels:
 dependencies:
   - python=3.10
   - lpython=0.20.0
-  - nodejs
-  - make
-  - cmake
+  - nodejs=18.12.1
+  - make=4.3
+  - cmake=3.25.2

--- a/environment.yml
+++ b/environment.yml
@@ -3,7 +3,7 @@ channels:
   - conda-forge
 dependencies:
   - python=3.10
-  - lpython=0.19.0
+  - lpython=0.20.0
   - nodejs
   - make
   - cmake

